app/controllers/orgs/rosters_controller.rb

```diff
@@ -228,6 +311,89 @@ def get_user_to_group_hash(grouping)
 
       mapping
     end
+
+    # Returns name of the linked google course to current organization (for syncing rosters)
+    def current_organization_google_course_name
+      return nil if current_organization.google_course_id.nil?
+      authorize_google_classroom
+      course = @google_classroom_service.get_course(current_organization.google_course_id)
+      return course.name unless course.nil?
+      nil
```

Comment:
This is a little odd, the fall through case of the method is the error case, and we bail early with the expected return type earlier in the method (320). People usually skim to the bottom of the method to see what it returns, so this is a little confusing.

Instead, we could replace 320 and 321 with `return unless course`, and `course.name` below.

 If we want to take it one step further, we can use the [safe navigation operator](https://stackoverflow.com/questions/36812647/what-does-ampersand-dot-mean-in-ruby) here and replace 320 and 321 with 

```ruby
course&.name
```

As an example:

```ruby
irb(main):004:0> (-5).abs
=> 5

irb(main):005:0> nil.abs
NoMethodError: undefined method `abs' for nil:NilClass
	from (irb):5
	from /usr/bin/irb:11:in `<main>'

irb(main):006:0> nil&.abs
=> nil

irb(main):007:0> (-5)&.abs
=> 5
```